base/strings/util.jl

```diff
@@ -109,6 +109,17 @@ used to implement specialized methods.
 !!! compat "Julia 1.5"
     The single argument `endswith(suffix)` requires at least Julia 1.5.
 
+# Examples
+```jldoctest
+julia> endswith_julia = endswith("Julia")
+(::Base.Fix2{typeof(endswith),String}) (generic function with 1 method)
```

Comment:
Maybe put a semicolon after this to avoid showing this really ugly type.

Reply:
Done